### DISCUSSION TITLE
Tordex True Launch Bar and ExplorerPatcher

### DISCUSSION THREAD

**Discussion by SoCu** [valinet/ExplorerPatcher#2364] [General]
Hi, I don't know what the problem may be, but I have been using the Tordex True Launch Bar application before trying ExplorerPatcher, and where you have a link to an application, when you passed the mouse over the icon it showed you a window with the last opened files, for example, Word showed you the last documents you have used, from where you could open the document you are most interested in, when installing ExplorerPatcher it no longer shows the last used documents. 

I have Windows 11 (22H2 - build 22621.2428) and ExplorerPatcher 22621.2428.59.1 although I don't think it has to do with this version, it has always happened to me.

This is the configuration I have now (pdf document):
https://drive.google.com/file/d/1N5OC2iFh2VXeqpJPELyZSUcMklZPjmRC/view?usp=sharing

I don't know if I have to configure something in ExplorerPatcher to show that popup window that are shown when I mouse over the application icons.

Thank you.

**Comment by pyrates999** [comment]
in File Explorer, untick **Disable the windows 11 context menu**.  And then restart file explorer.

Does that help?

**Comment by SoCu** [comment]
Hi pyrates999, yes I have tried it, but that does not solve anything, still do not appear those menus, and also does not appear the context menu when you click the right mouse button, something that also does not interest to disable.

But in the case of the Tordex True Launch Bar application, those menus still don't appear.

By the way, maybe I'm not doing it right, but I have disabled the context menu from:

Start > Run > gpedit.msc

**Comment by pyrates999** [comment]
Try uninstalling Tordex True Launch Bar, reboot, and then reinstall the latest EP.

**Comment by SoCu** [comment]
yes pyrates999, I tried that before writing here, and it didn't work.

**Comment by pyrates999** [comment]
I think you turned off file and folder history in windows explorer.  Try this to fix it: https://www.intowindows.com/fix-quick-access-not-showing-recent-files-in-windows-10/

**Comment by SoCu** [comment]
Everything was set up correctly, and I cleared the history, and it still doesn't show recent files.

Instead of showing recent files, it shows the path where the application is located.

https://i.imgur.com/p8HRJFa.png

**Comment by pyrates999** [comment]
You've got some software installed that already modifies the UI.  Please uninstall it as well.

**Comment by SoCu** [comment]
I don't have any installed, I used to use the Classic Start Menu, and everything worked fine.

**Reply by pyrates999** [reply]
You do.  Windows doesn't look like that normally.

**Comment by SoCu** [comment]
Something is wrong, I don't know if it has to do with Windows 11, I have uninstalled everything, reinstalled just the True Lanunch Bar, and it doesn't show recent files.

I have tried with StartAllBack, and it does not show recent files.

But how curious, I have uninstalled True Lanunch Bar, and I have left only the StartAllBack, and I check that the icons of the applications, if they show the recent files that were opened.

**Reply by pyrates999** [reply]
Try clearing all your folder and file history then by going into folder options in file explorer:

![image](https://github.com/valinet/ExplorerPatcher/assets/106135298/2022b797-9157-43e3-9abc-eec760b71f07)

**Comment by SoCu** [comment]
I have erased it, and it continues without showing the recent files.

The recent files, I have always had them, but when I installed Windows 11 they stopped showing, according to some comments, some update of Windows 11 changed some houses, maybe this is the fault of Windows 11, but what I find strange is that the StartAllBack application if it shows them, I speak only of the StartAllBack application, because if I install True Lanunch Bar are not shown in this application.

**Reply by pyrates999** [reply]
Once you clear your recent files and directories, you won't have any recent files and directories to show.

If you think it's your windows install, here's how to repair it.  Uninstall EP first though.

Then please run the following in an elevated command prompt or elevated powershell:

```
chkdsk C:
```

If it tells you there are errors, then run this:

```
chkdsk C: /F
```

Reboot to run it on boot.  Keep running that command until it reports no errors.  Then run this to repair windows:

```
sfc /scannow
dism /online /cleanup-image /scanhealth
dism /online /cleanup-image /restorehealth
dism /online /cleanup-image /checkhealth
dism /online /cleanup-image /analyzecomponentstore
dism /online /cleanup-image /startcomponentcleanup
```

Then reboot your computer.

**Comment by SoCu** [comment]
Sorry for not responding sooner, I was out, I checked, and there is nothing wrong.

**Reply by pyrates999** [reply]
This looks like it's an issue with your windows install, not EP.
